static/monorail-2.0.yaml

```diff
@@ -221,6 +247,24 @@ definitions:
             type: object
         type: object
     type: object
+  relations.2.0_Relation:
+    description: A key value set of relations.
+    properties:
+      containedBy:
+        description: The rack containing the node.
+        items:
+          type: string
+        minItems: 1
+        type: array
+        uniqueItems: true
+      contains:
+        description: An array of nodes contained by the rack.
+        items:
+          type: string
+        minItems: 1
+        type: array
+        uniqueItems: true
+    type: object
```

Comment:
@yyscamper I see swagger 2.0 supports only allOf in yaml. I don't see oneOf/anyOf working.